generators/app/index.js

```diff
@@ -9,49 +9,51 @@ module.exports = class extends Generator {
   }
 
   prompting() {
```

Comment:
you can change this method to async: `async promoting()` and then get the user info using `await`: `const githubUsername = await this.user.github.username()` so you don't need to nest everything inside `.then`